get_object/src/get_object.py

```diff
@@ -13,12 +13,30 @@ def get_object_data(objectId):
     return response['Item']
 
 def handler(event, context):
-    objectId = event ['queryStringParameters']['objectId']
-    item = get_object_data(objectId)
-    return {
-        'statusCode': 200,
-        'headers': {
-            'Content-Type': 'application/json'
-        },
-        'body': json.dumps(item)
-    }
+    try:
+        objectId = event ['queryStringParameters']['objectId']
```

Comment:
the first try block should be a validation. When queryStringParameters objectId is not available it should. return [Bad request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400)

This reduce also the nesting

Reply:
will be tracked in issue 71